### ISSUE TITLE
iOS 14 Native Compatibility

### ISSUE THREAD

**Issue by adammcarth** [adammcarth/react-native-segmented-picker#29]
```
[native] Error setting property 'options' of UIPicker with tag #519: Error setting property 'options' of UIPicker with tag #603: Exception thrown while executing UI block: *** __boundsFail: index 2 beyond bounds [0 .. 1]
```

I receive the above error when trying to open a Segmented Picker in `native` mode on iOS 14. May cause crashes in production - needs confirmation if this is a development only crash.

**Comment by adammcarth** [comment]
Oh yeah. Apple have definitely [revamped](https://developer.apple.com/design/human-interface-guidelines/ios/controls/pickers/) the UIPicker in iOS 14. This will need a fairly considered bug fix, as it is almost guaranteed to crash in production for iOS 14 users currently when the `native` option is used.

**Comment by adammcarth** [comment]
Fixed in `v2.0.2`.
